cmd/swarm/config.go

```diff
@@ -210,6 +210,18 @@ func flagsOverride(currentConfig *bzzapi.Config, ctx *cli.Context) *bzzapi.Confi
 		currentConfig.SwapEnabled = true
 	}
 
+	if swapBackendURL := ctx.GlobalString(SwarmSwapBackendURLFlag.Name); swapBackendURL != "" {
+		currentConfig.SwapBackendURL = swapBackendURL
+	}
+
+	if paymentThreshold := ctx.GlobalUint64(SwarmSwapPaymentThresholdFlag.Name); paymentThreshold != 0 {
+		currentConfig.SwapPaymentThreshold = paymentThreshold
+	}
+
+	if disconnectThreshold := ctx.GlobalUint64(SwarmSwapDisconnectThresholdFlag.Name); disconnectThreshold != 0 {
```

Comment:
just to check: it's valid for the user to provide just _one_ of the thresholds and not the other, correct?

Reply:
Yes, this must be possible.

Below is my own understanding. This paymentThreshold SWIP is not written and will give more clarity on the subject:

The disconnect threshold can always be entirely set by users, although we should warn them if it is set below the value which is quoted by the payment threshold oracle. 
The payment threshold will be quoted by an oracle and for this oracle, there is a default address set which can be overridden. We should, although, not encourage users to do this as, as it is possible that, by referring a different oracle they will pay too soon or too late, leading to disconnects.

Reply:
i see; when you refer to the payment threshold SWIP—is this about an oracle to provide this information?

Reply:
Yes! linking: https://swarmresear.ch/t/adjusting-payment-thresholds/42